libs/application/templates/estate/src/fields/RealEstateRepeater/index.tsx

```diff
@@ -105,7 +105,7 @@ export const RealEstateRepeater: FC<FieldBaseProps<Answers>> = ({
                   disabled={!asset.enabled}
                   backgroundColor="blue"
                   placeholder="0 kr."
-                  defaultValue={(asset as any).marketValue}
```

Comment:
Can you try seeing if the type `EstateAsset` is importable to this context, becoming
```typescript
defaultValue={(asset as EstateAsset)?.marketValue}
```

Reply:
Lol, I just commented on a removed line. Ignore me :>